examples/extensions/horizontal-pod-autoscaling/README.md

```diff
@@ -33,20 +33,14 @@ Documentation for other releases can be found at
 
 # Horizontal Pod Autoscaler
 
-Horizontal pod autoscaling is an experimental feature in Kubernetes 1.1.
-It allows the number of pods in a replication controller or deployment to scale automatically based on observed CPU or memory usage.
-<b>Please note that the current API is tentative and will be subject to change before a stable version is released.</b>
+Horizontal pod autoscaling is a beta feature in Kubernetes 1.1.
+It allows the number of pods in a replication controller or deployment to scale automatically based on observed CPU usage.
```

Comment:
I would mention that in a future version it will support scaling based on other metrics.